COLLADAMaya/scripts/COLLADAMaya.vcxproj

```diff
@@ -1730,7 +1730,7 @@ copy "$(ProjectDir)*.mel" "%25MAYA_PATH2014_X64%25\scripts\others"
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='DebugPlugin2015|x64'">
     <PreBuildEvent>
-      <Command>call create_version_info_h.bat $(PLATFORM) $(CONFIGURATION)</Command>
+      <Command>call modify_version_info_h.bat</Command>
```

Comment:
So you prefer i remove this script from the "prebuildEvent" in visual studio and
End user will run the script as needed